server/routerlicious/packages/lambdas-driver/src/kafka-service/checkpointManager.ts

```diff
@@ -80,7 +90,15 @@ export class CheckpointManager {
 			},
 			// eslint-disable-next-line @typescript-eslint/promise-function-async
 			(error) => {
-				// Enter an error state on any commit error
+				if (error.name === "PendingCommitError") {
+					Lumberjack.info(`Skipping checkpoint since ${error.message}`, {
+						queuedMessageOffset: queuedMessage.offset,
+						queuedMessagePartition: queuedMessage.partition,
+					});
+					this.checkpointing = false;
```

Comment:
Just for my own understanding, what the significance of setting this here?
this.checkpointing = false;